applications/ila-cookie-banner/RELEASE.md

```diff
@@ -0,0 +1,94 @@
+# Cookie Banner Release Process
+
+This is a guide for installing the a new release of the campus cookie banner to the content delivery network (CDN) server S3 bucket.
+
+The target audience for this document is the team that maintains the CDN that serves the shared cookie banner.
+
+## Stage the New Release
+
+### Step 1. Create the new release folder.
+
+	Create a new folder, named for the versioned release. For example, if `1.0.0`:
+
+		https://onetrust.techservices.illinois.edu/1.0.0
+		
+### Step 2. Get the latest cookie banner zip file from GitHub Actions.
+
+	+ Visit https://github.com/app-illinois/Design-Resources/actions
+	+ Click on `Cookie banner zip`
+	+ Click on the the latest successful run - the top one with a green icon.
+
+	> Tip: Failed runs are not expected. Contact the development team if the most recent `Cookie banner zip` GitHub action failed (indicated by a red icon).
+
+	+ Find and click `cookie-banner` under the `Artifacts` section of the `GitHub Actions Cookie banner zip` page.
+	+ Find the Zip file in your browser downloads directory.
```

Comment:
It seems like the formatting for this list may be incorrect. 
When I preview in this view, the "plus-signs" and "carrot" do not display with Markdown styling.